bot/cogs/bot.py

```diff
@@ -60,6 +65,36 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.channel.id in self.code_block_channels:
+            if self.code_block_channels[msg.channel.id]-time.time() > 300:
+                if msg.content.count("\n") >= 3:
+                    try:
+                        tree = ast.parse(msg.content)
+
+                        # Attempts to parse the message into an AST node.
+                        # Invalid Python code will raise a SyntaxError.
+                        if not all(isinstance(node, ast.Expr) for node in tree.body):
+
+                            # We don't want multiple lines of single words,
```

Comment:
Can I suggest having:
`# Multiple lines of single words could be interpreted as expressions.`
`# This check is to avoid all nodes being parsed as expressions.`
`# (e.g. words over multiple lines)`